packages/dashboard/src/store/actions/pasteWidgets/index.ts

```diff
@@ -23,12 +22,18 @@ export const pasteWidgets = (state: DashboardState, action: PasteWidgetsAction):
 
   const cellSize = state.grid.cellSize;
   const copyGroup = state.copiedWidgets;
+  const xgrid = state.grid.width;
```

Comment:
i think renaming this to width or gridWidth would make more sense ?

Reply:
Resolved